RHEL9 feed installation instructions (#219)
redis-stack/redis-stack-docs: docs/stack/install/install-stack/linux.md

sudo yum install redis-stack-server
{{< / highlight >}}

> Please remember to replace rhel7 with either rhel8 or rhel9, based on your operating system.

Reviewer: ```suggestion
Be sure to replace _rhel7_ with either _rhel8_ or _rhel9_ as appropriate.
```
